client-v2/src/main/java/com/clickhouse/client/api/Client.java

```diff
@@ -2186,8 +2160,28 @@ public void updateClientName(String name) {
         this.configuration.put(ClientConfigProperties.CLIENT_NAME.getKey(), name);
     }
 
+    public static final String clientVersion = getPackageVersion();
+    public static final String CLIENT_USER_AGENT = "clickhouse-java-v2/";
+
     private Collection<String> unmodifiableDbRolesView = Collections.emptyList();
 
+    private static String getPackageVersion() {
```

Comment:
This code is duplicated. Can we use some utility function

Reply:
I thought about it, but it is in different packages. 
We can have it in old client project because it is used everywhere. Will see what I can do.